src/services/auth-provider.js

```diff
@@ -1,38 +1,39 @@
-const loginUrl = 'https://comments-microservice.herokuapp.com/v1/msadmins/login';
+const loginUrl =
+  "https://comments-microservice.herokuapp.com/v1/msadmins/login";
 
 export default {
   login: async ({ username, password }) => {
     const request = new Request(loginUrl, {
-      method: 'POST',
+      method: "POST",
       body: JSON.stringify({ email: username, password }),
-      headers: { 'Content-Type': 'application/json' },
+      headers: { "Content-Type": "application/json" },
     });
     return fetch(request)
-      .then(res => {
+      .then((res) => {
         if (res.status < 200 || res.status >= 300) {
           throw new Error(res.statusText);
         }
         return res.json();
       })
       .then(({ data }) => {
-        localStorage.setItem('systemToken', data.systemToken);
+        localStorage.setItem("systemToken", data.systemToken);
       });
   },
   logout: () => {
-    localStorage.removeItem('systemToken');
+    localStorage.removeItem("systemToken");
     return Promise.resolve();
   },
   checkError: (error) => {
     if (error.status === 401 || error.status === 403) {
```

Comment:
This line causes a `TypeError: Cannot read property 'status' of undefined` whenever I, as a user, try and save an updated settings page. 

NOTE: This is url on the browser address bar when on the edit settings page: `http://localhost:3000/comment-microapi-admin-dashboard#/Settings/undefined`.

![Comment Dashboard - Google Chrome 2020_07_22 00_43_28 (2)](https://user-images.githubusercontent.com/45206423/88114865-20eacf80-cbb5-11ea-9b81-7656d4547a78.png)

Reply:
even when I added an `id` field, rendering settings page still generate this error.. 
![image](https://user-images.githubusercontent.com/17856665/88151204-4fd76480-cbfa-11ea-83fa-94bb4921cadb.png)

Reply:
so I've removed edit option from UI and will revisit later..
possibly, we can add all pages to get a clearer idea of fixes and then come back for any fixes left..?
this way we make ui progress

Reply:
my guess is that I need to use a custom menu for the settings page but will leave this for now

Reply:
this error shows why every record must have an `id` field..

Reply:
We can revisit as I don't have a full understanding of the error either.